src/views/phase-selection.module.css

```diff
@@ -0,0 +1,60 @@
+.phasePage {
+	display: flex;
+	flex-direction: column;
+	flex: 1;
+	justify-content: center;
+	gap: 60px;
+	overflow: hidden;
+	background-image: url('../assets/images/grave-robber.png');
+	background-repeat: no-repeat;
+	background-position: center;
+}
+
+.line {
+	background-color: white;
+	width: 100%;
+	height: 1px;
+	transform: rotate(-30deg);
+}
+
+.phaseType {
+	display: flex;
+	flex-direction: column;
+	gap: 12px;
+}
+
+.phaseType h1 {
+	margin-bottom: -20px;
+}
+
+.dawnPhase {
+	margin-right: auto;
+	margin-left: max((100vw - 377px) * .2, 0px);
+}
+
+.nightPhase {
+	margin-left: auto;
+	margin-right: max((100vw - 377px) * .2, 0px);
+}
+
+
+
+.header {
+	font-size: 120px;
+}
+
+.arrowIcon {
+	transform: rotate(-90deg);
+}
+
+
+.constable {
+	display: flex;
+	align-items: center;
+	font-family: IM FELL English SC;
+	font-size: 18px;
+	letter-spacing: 2px;
+    color: #F1ECCB;
```

Comment:
Same issue here. But in this case you were able to add spaces instead. Just a plug for getting this fixed in the future